packages/design-system-docs/src/pages/startup/components.md

```diff
@@ -55,9 +55,9 @@ ReactDOM.render(
 
 See [React component documentation]({{root}}/components/button/#components.button.react) for more react examples, guidance, and detailed prop documentation.
```

Comment:
I'm wondering if we change the wording here to something like this and remove the link to the button page?

> React prop documentation, examples, and guidance can be found on component pages.